wiki/UI-component/Video.md

```diff
@@ -39,6 +39,147 @@ The [[Video]] component utilizes the embeddable Kaltura player <sup>[[r2](#resou
 
 <hr width="40%" />
 
+## 1. Data & content
+
+The video component intakes a Kaltura `videoId` and fetches the following data to be rendered on page as content. All of these content can be customized except the video duration.
+
+See more details at [Kaltura Player API](http://player.kaltura.com/docs/api).
+
+| Data name | Type | Allows override |
+| --------- | ---- | --------------- |
+| Name | Text | Yes |
+| Description | Text | Yes |
+| Duration | Number | No |
+| Poster image | Image | Yes |
+
+## 2. Rendering & behavior
+
+Video is used in the following components. Depending on the visual design and content requirement of the final component rendered, the above data is used differently.
+
+### Video
+
+This is the default rendering of video. It is used for embedding a video directly on page or inline.
```

Comment:
1. Cross link when possible. Even if it's back to the same page.
2. It's less of embedding a video, and more of embedding the Kaltura player
3. Nit picky thing, but inline is the same as directly in the page in my opinion.

```suggestion
This is the default rendering of [[Video]]. It is used for embedding a Kaltura player <sup>[[r2](#resources)]</sup> directly in the page.
```